pkgs/games/forge-mtg/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, bzip2, fetchurl, makeWrapper, coreutils, gnused, openjdk, stdenv }:
+stdenv.mkDerivation rec {
+  pname = "forge-mtg";
+  version = "1.6.53";
+
+  src = fetchurl {
+    url = "https://releases.cardforge.org/forge/forge-gui-desktop/${version}/forge-gui-desktop-${version}.tar.bz2";
+    sha256 = "sha256-rRb4PevxKMw4GN0q1oMRRsXQfgZhrNrJ3S++CuctRvA=";
+  };
+  sourceRoot = ".";
+
+  dontBuild = true;
```

Comment:
Is there a reason to use the prebuilt release instead of pulling the source? It seems like a fairly standard maven build.

Reply:
Not directly. I had some bad experiences attempting to package Gradle applications and packaging the byte-code still seems to be the prevalent mode of packaging Java applications in nixpkgs, so I went with the upstream release first.

But since you suggested it, I also gave mvn2nix+mavenBuild a quick test-drive, but we run into two issues:

1. Maven central frequently sends a TCP reset when dependencies are fetched using curl (via `fetchurl`), so the build took about 10 retries before all dependencies had been fetched.
2. Building the resulting derivation fails due to missing dependencies (perhaps related to the multi-project POM or a similar limitation of mvn2nix).

Next, I gave the more classical approach of running `mvn package -Dmaven.repo.local=$out` a shot. Here the build will fail due to the project depending on launch4j. launch4j tries to run a native binary as part of the package phase and of course that binary is linked against some libraries in `/usr/lib`. However, once we patch out launch4j, things seem to work as expected.

Could you give this another look if time permits?